package-lock.json

```diff
@@ -1898,6 +1898,12 @@
                     "resolved": "https://registry.npmjs.org/source-map/-/source-map-0.5.7.tgz",
                     "integrity": "sha1-igOdLRAh0i0eoUyA2OpGi6LvP8w=",
                     "dev": true
+                },
+                "typescript": {
```

Comment:
This appears to be due to @angular-devkit/core. It would be nice if we can dedupe typescript to have only one instance, or there tends to be build issues. Perhaps we can bump the devkit packages too?